source/_integrations/alexa.intent.markdown

```diff
@@ -17,9 +17,9 @@ The built-in Alexa integration allows you to integrate Home Assistant into Alexa
 
 ### Requirements
 
-- Amazon Developer Account. You can sign on [here](https://developer.amazon.com).
-- An [AWS account](https://aws.amazon.com/free/) is need if you want to use Alexa Custom Skill API. Part of your Alexa Custom Skill will be hosted on [AWS Lambda](https://aws.amazon.com/lambda/pricing/). However you don't need worry the cost, AWS Lambda allow free to use up to 1 millions requests and 1GB outbound data transfer per month.
-- The Alexa Custom Skill API also needs your Home Assistant instance to be accessible from the internet via HTTPS on port 443 using a certificate signed by [an Amazon approved certificate authority](https://ccadb-public.secure.force.com/mozilla/IncludedCACertificateReport), this is so account linking can take place. Read more on [our blog](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) about how to set up encryption for Home Assistant. When running Home Assistant using the [Duck DNS](/addons/duckdns/) add-on is the easiest method.
+- Amazon Developer Account. You can sign in [here](https://developer.amazon.com).
```

Comment:
This change is incorrect. This is the requirements listing, and tell you where to get an account. You sign up or on for a new account, but you sign-in into an existing account.

Please revert this change.

Reply:
I was under the wrong impression you could just log in your current Amazon account. My bad.